terraform/api_gateway.tf.json.template.py

```diff
@@ -157,28 +169,23 @@ def for_domain(cls, domain):
                     } for i, domain in enumerate(lambda_.domains)
                 }
             },
-            **(
-                {
-                    "aws_cloudwatch_log_group": {
-                        lambda_.name: {
-                            "name": "/aws/apigateway/" + config.qualified_resource_name(lambda_.name),
-                            "retention_in_days": 1827,
-                            "provisioner": {
-                                "local-exec": {
-                                    "command": ' '.join(map(shlex.quote, [
-                                        "python",
-                                        config.project_root + "/scripts/log_api_gateway.py",
-                                        "${module.chalice_%s.RestAPIId}" % lambda_.name,
-                                        config.deployment_stage,
-                                        "${aws_cloudwatch_log_group.%s.arn}" % lambda_.name
-                                    ]))
-                                }
-                            }
+            "aws_cloudwatch_log_group": {
+                lambda_.name: {
+                    "name": "/aws/apigateway/" + config.qualified_resource_name(lambda_.name),
+                    "retention_in_days": 1827,
+                    "provisioner": {
+                        "local-exec": {
+                            "command": ' '.join(map(shlex.quote, [
+                                "python",
+                                config.project_root + "/scripts/log_api_gateway.py",
+                                "${module.chalice_%s.RestAPIId}" % lambda_.name,
+                                config.deployment_stage,
+                                "${aws_cloudwatch_log_group.%s.arn}" % lambda_.name
+                            ]))
                         }
                     }
-                } if config.enable_monitoring else {
```

Comment:
Why this change? Did you test deploying this with AZUL_ENABLE_MONITORING turned off?

Reply:
CloudWatch logs should be enabled at all times. Additionally, enabling `AZUL_ENABLE_MONITORING` sets up other monitoring resources (health checks, alarms, etc.) which was beyond the scope of this ticket.